scipy/signal/filter_design.py

```diff
@@ -4843,6 +4845,200 @@ def _design_notch_peak_filter(w0, Q, ftype, fs=2.0):
     return b, a
 
 
+def _hz_to_erb(hz):
```

Comment:
this entire block down to line `5040` was excluded in my more conservative test merge--not sure if the private function is actually harmful, but just noting that difference